playground/dist/index.html

```diff
@@ -114,12 +130,23 @@ <h1 class="vqb__drop-zone__message">
       <button class="open-code__button" v-show="isCodeOpened" @click="hideCode">
         Hide code
       </button>
-      <pre v-show="isCodeOpened"><code>{{ code }}</code></pre>
-      <div class="warning-message" v-if="backendWarnings.length" v-for="warning in backendWarnings">
-        <i class="fas fa-exclamation-triangle"></i> {{ warning.message }}
-      </div>
-      <div class="error-message" v-if="backendErrors.length" v-for="error in backendErrors">
-        <i class="fas fa-exclamation-triangle"></i> {{ error.message }}
+      <div v-if="isCodeOpened" class="debug-panel">
+        <div class="warning-message" v-if="backendWarnings.length" v-for="warning in backendWarnings">
+          <i class="fas fa-exclamation-triangle"></i> {{ warning.message }}
+        </div>
+        <div class="error-message" v-if="backendErrors.length" v-for="error in backendErrors">
+          <i class="fas fa-exclamation-triangle"></i> {{ error.message }}
+        </div>
```

Comment:
I'm not sure we want to display that only when code panel is opened. This should appear to users anytime it's needed!

Instead, I suggest we remove the default warning (over 50 lines) which was a nice testing case but shouldn't appear in the playground.